openh264/openh264_make_archive

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#
+# Archive last build of openh264 to :POST and make a zip file from its contents
+#
+# Contact openSUSE Release Team for more information or contacts
+# https://en.opensuse.org/openSUSE:Release_team
+#
+# More details about OpenH264 in openSUSE at https://en.opensuse.org/OpenH264"
+
+PROJ="openSUSE:Factory:openh264:POST"
+
+echo
+echo "This tool will make zip archive with binaries from $PROJ"
+echo "This zip archive can be then sent over to Cisco for extraction at ciscobinary.openh264.org."
+echo
+echo "Please make sure that you've made snapshot of data (release_to_post.sh) prior proceeding."
+echo
+echo "Press Enter to proceed or ctrl+c to cancel."
+read
+
+tempdr=`mktemp -d /tmp/cisco-binary-opensuse-XXXX`
+bindr="$tempdr/cisco-openh264-rpms-opensuse"
+checkoutdr="$tempdr/checkout"
+mkdir $bindr
+mkdir $checkoutdr
+
+pushd $checkoutdr
+# Checkout only metadata to speed it up
+osc -A https://api.opensuse.org co -M $PROJ
+pushd $PROJ:POST
+
+# Get binaries for all repos / arches
+for pkg in *; do
+  pushd $pkg
+  for repo in `osc repos | awk '{ print $1 }' | sort | uniq`; do
+    osc getbinaries $repo -d $bindr
+  done
+  popd
+done
+
+pushd $tempdr
+# make zip only of rpms, skip logs and metadata
+archive="`basename $bindr`"
```

Comment:
It's a bit of a personal style choice, but I prefer `$()` over backticks

Reply:
I'd suggest adding DATE to the archive name; making it easier to identify the most recent zip file

Reply:
Makes sense! Let me add that

Reply:
YYYY-MM-DD added I expect no fights aroud format :-)